packages/editor/src/components/post-last-revision/index.js

```diff
@@ -31,13 +31,13 @@ function LastRevision() {
 				} ) }
 				className="editor-post-last-revision__title"
 				icon={ backup }
-			>
-				{ sprintf(
-					/* translators: %d: number of revisions */
-					_n( '%d Revision', '%d Revisions', revisionsCount ),
+				iconPosition="right"
+				text={ sprintf(
+					/* translators: %s: number of revisions */
+					__( 'Revisions (%s)' ),
 					revisionsCount
 				) }
-			</Button>
+			></Button>
```

Comment:
```suggestion
			/>
```

Since Button has no children, it should be possible to simplify it like this